src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Avatar from './Avatar';
+import { UserProfile } from 'services/user';
+
+import styles from './Layout.module.css';
+
+interface ComponentProps {
+  show: boolean;
+  setShow: React.Dispatch<React.SetStateAction<boolean>>;
+  user: UserProfile;
+}
+
+const Sidebar = ({ show, setShow, user }: ComponentProps) => {
+  return (
+    <>
+      {show && (
+        <div
+          className="fixed top-0 right-0 h-screen w-screen"
+          data-testid="sidebar-overlay"
+          onClick={() => setShow(false)}
+        />
+      )}
+      <aside className={styles.sidebar} aria-hidden={!show}>
+        <section className="flex justify-between items-center">
+          <div className="font-extrabold text-white">{user.email}</div>
```

Comment:
Should we set the `font-size` and `line-height`?

![Screen Shot 2565-05-11 at 11 40 04](https://user-images.githubusercontent.com/1772999/167770413-c333b0f9-9208-4ebc-8891-fce33bd4623d.png)

Reply:
Updated in [b759945](https://github.com/hanam1ni/react-surveyor/pull/60/commits/b759945159bbd42134f9e36837fbe450683ac6e9)